app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -19,6 +20,11 @@ def show
     end
   end
 
+  def decision_notice
+    @planning_application = current_local_authority.planning_applications.where(id: params[:id]).first
+    @disable_headers = true
```

Comment:
It could be just me, but `disable_headers` momentarily made me think something was being done to the HTTP headers rather than the layout